### PR TITLE
Preprocessing workflow for PCNA dataset

### PR DESCRIPTION
**Pull request by ritvikvasan**
Currently, the preprocessing only works for the variance dataset. 

This PR 
1. Adds a separate snakefile (Snakefile_pcna) for running the preprocessing for PCNA dataset. 
2. Adds the config group which has the set of arguments needed to run the PCNA dataset correctly
3. Removes many unused components and arguments of different steps
4. Runs pre-commit on all files

To test, here is a local subset copy of the pcna dataset - `/allen/aics/modeling/ritvik/projects/pcna_test.csv`

Update `config.yaml` in the `config_pcna` folder with this path and run

`snakemake -s Snakefile_pcna --cores all`

### CODE REVIEW

**Comment by ritvikvasan** [comment]
actually, I still need to work on this. Please don't review yet

**Comment by ritvikvasan** [comment]
@pgarrison @fatwir @vianamp  this PR is ready for review

**Comment by fatwir** [comment]
Before running the script, please ensure the following:

1. **Set the `TMPDIR`:**
   You need to set the `TMPDIR` environment variable, as the Snakefile requires a temporary directory. You can do this by executing:
   ```bash
   export TMPDIR=/path/to/your/tmpdir
   ```

2. **Set the `output_dir`:**
   Additionally, you must specify the `output_dir` required by [this](https://github.com/AllenCell/benchmarking_representations/blob/pcna_preprocessing/subpackages/image_preprocessing/config_pcna/config.yaml) config file.

Once these variables are correctly set, the scripts should work as expected!

**Review by fatwir — APPROVED** [review]

File: subpackages/image_preprocessing/Snakefile [hunk 1]
```diff
@@ -54,15 +49,3 @@ rule register:
         os.path.join(cfg.output_dir, "register/manifest.parquet")
     shell:
         "python scripts/run_step.py step=register ++input={input}"
-
-rule delete_temps:
-    shell:
-        """
-        wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
```
Comment (pgarrison) [inline subpackages/image_preprocessing/Snakefile:61]: Didn't know this was here! Glad to see it removed

**Review by pgarrison — APPROVED** [review]

**Pull request merged**